src/test/test-models/good/compound-assign/indexed_expr_divide_equals.stan

```diff
@@ -3,5 +3,7 @@ transformed data {
   a /= 5.0;
   print("a: ", a);
   a[1, ] /= 5.0;
+  a[1, 1:2] /= 5.0;
+  a[, 1:2] /= 5.0;
   print("r1 div 5: ", a);
```

Comment:
Are these printed values checked anywhere? If so this change will probably fail the test.

Reply:
cc @bob-carpenter but I'm pretty sure the test here just checks if the stan programs will compile